Makefile

```diff
@@ -50,16 +50,20 @@ lint-js:
 	@yarn eslint --ext js --report-unused-disable-directives "**/*.js"
 	@yarn prettier --check "**/*.{js,ts,md}"
 lint-python:
-	@python3 -m flake8 ./
+	@python3 -m flake8p ./
```

Comment:
Note the plugin for using a `pyproject.toml` file requires a different command. (see [here](https://pypi.org/project/Flake8-pyproject/)).

Reply:
> For compatibility with earlier versions of this package, and perhaps extra reliability in terms of possible future breakage of the plug-in hook, the package also provides a flake8p command that could be called alternatively to lint the code.

The way I read this, we can keep using the normal `flake8` command. Is that not the case?